mst/user-management.md

```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
 
 </procedure>
 
+## Multi-factor user authentication
+
+You can use multi-factor authentication (MFA) to log in to your Managed Service
+for TimescaleDB account. This requires an authentication code, provided by the
+Google Authenticator app on your mobile device, to be able to log in to your
+Managed Service for TimescaleDB account.
+
+You can see which authentication method is in use by each member of your Managed
+Service for TimescaleDB project. From the dashboard, navigate to the `Members`
+section. Each member is listed in the table with an authentication method of
+either `Password`, or `Two-Factor`.
+
+Before you begin, install the Google Authenticator app on your mobile device.
+For more information, and installation instructions, see
+[the Google Authenticator documentation][install-google-authenticator].
+
+<procedure>
+
+### Configuring multi-factor authentication
+
+1.  [Log in to your account][mst-login].
+1.  Click the `User information` icon in the top-right of the dashboard to go to
+    the `User profile` section.
+1.  In the `Authentication` tab, toggle `Two-factor authentication` to
+    `Enabled`, and enter your password.
+1.  On your mobile device, open the Google Authenticator app, tap `+` and select
+    `Scan a QR code`.
+1.  On your mobile device, scan the QR code provided by Managed Service for
+    TimescaleDB.
+1.  In your Managed Service for TimescaleDB dashboard, enter the confirmation
+    code provided by the Google Authenticator app, and click
+    `Enable Two-Factor Auth`.
+
+</procedure>
+
+<highlight type="warning">
+If you have lost access to the mobile device uses for your multi-factor
+authentication, you cannot sign in to your Managed Service for TimescaleDB
+account. To regain access to your account, on the login screen, click
+`Forgot password?` and follow the step to reset your password. When you have
+regained access to your account, reconfigure multi-factor authentication.
+</highlight>
+
+## User authentication tokens
+
+Every time a registered user logs in, Managed Service for TimescaleDB creates a
+new authentication token. This occurs for login events using the portal, and
+using the API. By default, authentication tokens expire after 30 days, but the
+expiry date is adjusted every time the token is used. This means that tokens can
+be used indefinitely, if the user logs in at least every thirty days.
```

Comment:
Not terribly bothered by whether we spell out `thirty` or write `30`, but we should make it consistent within this paragraph.

Reply:
oof, good spot!

```suggestion
be used indefinitely, if the user logs in at least every 30 days.
```